src/libcore/tests/num/int_log.rs

```diff
@@ -0,0 +1,64 @@
+#[test]
+fn log() {
+    assert_eq!(999u32.log(10), Some(2));
+    assert_eq!(1000u32.log(10), Some(3));
+    assert_eq!(555u32.log(13), Some(2));
+    assert_eq!(63u32.log(4), Some(2));
+    assert_eq!(64u32.log(4), Some(3));
+    assert_eq!(10460353203u64.log(3), Some(21));
+    assert_eq!(10460353202u64.log(3), Some(20));
+    assert_eq!(147808829414345923316083210206383297601u128.log(3), Some(80));
+    assert_eq!(147808829414345923316083210206383297600u128.log(3), Some(79));
+    assert_eq!(22528399544939174411840147874772641u128.log(19683), Some(8));
+    assert_eq!(22528399544939174411840147874772631i128.log(19683), Some(7));
+
+    for i in std::i16::MIN..0 {
+        assert_eq!(i.log(4), None);
+    }
+    for i in 1..std::i16::MAX {
+        assert_eq!(Some((i as f32).log(13.0) as i16), i.log(13));
```

Comment:
Rust explicitly uses `rhs` and `lhs` in their terminology because the testing framework does not prescribe which order arguments should come in. Either order is permitted.

Reply:
Reopening because even without an order prescribed, switching back and forth from line to line within the same test function is disorienting. These tests would be easier to read if the `i.log(_)` call being tested were on the same side.

Reply:
@dtolnay ah I missed they weren't consistent in the same file; agreed!